components/lib/hooks/useMergeProps.js

```diff
@@ -10,7 +10,7 @@ export const useMergeProps = () => {
 
     return (...props) => {
         const options = {
-            ...(context?.ptOptions?.classNameMergeFunction && { classNameMergeFunction: context.classNameMergeFunction })
+            ...(context?.ptOptions?.classNameMergeFunction && { classNameMergeFunction: context.ptOptions.classNameMergeFunction })
```

Comment:
```suggestion
            ...(context?.ptOptions?.classNameMergeFunction)
```

Should it just be the above?  I am not sure what its appending classNameMergeFunction to itself again now?

Reply:
Should be syntactically equivalent. Let me check.